frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -240,9 +240,11 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
         content: confirmMessage,
         confirmButtonClass: 'btn-info',
         cancelButtonClass: 'btn-danger',
-        confirm: function() {
-            confirmCallback();
-        }
+        buttons: {
```

Comment:
This creates a confirm dialog with no option to cancel. I see you have bumped the version of jquery-confirm, but didn't sufficiently adapt to its new API. Please make sure to restore the functionality.